kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KafkaNativeContainerIT.java

```diff
@@ -271,4 +310,14 @@ void testKraftClusterOneController() throws Exception {
         }
     }
 
+    MountableFile classpathResourceWithStringsReplaced(String resourceName, Path tempDir, Map<String, String> replacements) throws IOException {
```

Comment:
I'd change the `withServerPropertes` method to take a `Transferable` instead. That way you don't need to do this trick, You can just build a `Transferable.of` with the string and pass that.